### PR TITLE
[#2026] Force drag and resizing to be limited to parent element. Removed dead code for old media view

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2026-constrain-draggable-area-for-plugins-to-video-parent-div

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2dfae13e9f2008e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/3b6e815e9a73c8c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2dfae13e9f2008e/output.txt

Total script time: 0.51 mins

Server is running on http://pr810.butyr.org/templates/basic/
Run the Test Suite: http://pr810.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3b6e815e9a73c8c/output.txt

Total script time: 0.78 mins

File: src/core/media.js [hunk 1]
```diff
@@ -93,9 +89,6 @@
               _this.dispatch( "mediaready" );
             },
             constructing: function(){
```
Comment (cadecairos) [inline src/core/media.js]: is this Fn even necessary?

File: templates/assets/plugins/wikipedia/popcorn.wikipedia.js [hunk 2]
```diff
@@ -55,10 +51,10 @@
...
 
       _container.classList.add( "wikipedia-inner-container" );
-      _container.style.width = normalize( options.width || _manifestOpts.width[ "default" ], 20, 100 ) + "%";
-      _container.style.height = normalize( options.height || _manifestOpts.height[ "default" ], 30, 100 ) + "%";
-      _container.style.top = normalize( options.top || _manifestOpts.top[ "default" ], 0, 70 ) + "%";
-      _container.style.left = normalize( options.left || _manifestOpts.left[ "default" ], 0, 80 ) + "%";
+      _container.style.width = options.width || _manifestOpts.width[ "default" ] + "%";
+      _container.style.height = options.height || _manifestOpts.height[ "default" ] + "%";
+      _container.style.top = options.top || _manifestOpts.top[ "default" ] + "%";
+      _container.style.left = options.left || _manifestOpts.left[ "default" ] + "%";
```
Comment (cadecairos) [inline templates/assets/plugins/wikipedia/popcorn.wikipedia.js]: the above lines need brackets:

`_container.style.width = ( options.width || _manifestOpts.width[ "default" ] ) + "%";

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e850ce67301ef0c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/0a0efc60f3031db/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e850ce67301ef0c/output.txt

Total script time: 0.84 mins

Server is running on http://pr810.butyr.org/templates/basic/
Run the Test Suite: http://pr810.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0a0efc60f3031db/output.txt

Total script time: 1.14 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/86e7cc7a299bde5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/4f441a4013ff736/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/86e7cc7a299bde5/output.txt

Total script time: 0.77 mins

Server is running on http://pr810.butyr.org/templates/basic/
Run the Test Suite: http://pr810.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/4f441a4013ff736/output.txt

Total script time: 1.03 mins

**Pull request merged**